packages/sdk/chainDeploy/helpers/liquidators/fuseSafeLiquidator.ts

```diff
@@ -76,6 +76,17 @@ export const configureFuseSafeLiquidator = async ({
     }
   }
 
+  for (const address in chainIdToConfig[chainId].fundingStrategies) {
+    const [redemptionStrategyType] = chainIdToConfig[chainId].fundingStrategies[address];
```

Comment:
`fundingStrategies[address]` returns `[FundingStrategyContract, string]` right? I hope the destructuring of `const [redemptionStrategyType] =` works fine, just wondering if it doesn't need smt like `const [redemptionStrategyType, ] =`